modules/thorvg/image_loader_thorvg.cpp

```diff
@@ -0,0 +1,114 @@
+/*************************************************************************/
+/*  image_loader_thorvg.cpp                                              */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2021 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2021 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "image_loader_thorvg.h"
+#include "core/error/error_macros.h"
+#include "core/math/vector2.h"
+#include "thorvg.h"
+#include <stdint.h>
+#include <memory>
+
+#include "thirdparty/thorvg/inc/thorvg.h"
```

Comment:
Please review https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes

Proper style here would be:
```suggestion
#include "image_loader_thorvg.h"

#include "core/error/error_macros.h"
#include "core/math/vector2.h"

#include <memory>
#include <stdint.h>
#include <thorvg.h>
```

`thirdparty/thorvg/inc/thorvg.h` is redundant and shouldn't be needed if you get the CPPPATH right.